arangod/Cluster/ClusterTypes.h

```diff
@@ -66,10 +67,19 @@ class RebootId {
     return RebootId{std::numeric_limits<decltype(_value)>::max()};
   }
 
+  std::ostream& print(std::ostream& o) const {
+    o << _value;
+    return o;
+  }
+
  private:
   uint64_t _value;
 };
 
 }  // namespace arangodb
 
+inline std::ostream& operator<< (std::ostream& o, arangodb::RebootId const& r) {
+  return r.print(o);
+}
```

Comment:
@kvahed A side note. This is a central header file, and including `<iostream>` here will thus incur quite a compile time overhead. I'd recommend to include `<iosfwd>` only, and move the `print` and `operator<<` implementations in a corresponding `.cpp` file. Or even move it to a completely different file, that has to be included separately.